search-app/src/cmr/search/api/association.clj

```diff
@@ -57,6 +57,18 @@
     400
     200))
 
+(defmulti valid-association-args-count?
```

Comment:
For things this simple, we should just use a single function with `if` statement.